packages/shopware-6-client/src/services/categoryService.ts

```diff
@@ -1,16 +1,60 @@
-import { getCategoryEndpoint } from "../endpoints";
+import { Category } from "../interfaces/models/content/category/Category";
+import { getCategoryEndpoint, getCategoryDetailsEndpoint } from "../endpoints";
+import { ParamsConverter } from "../helpers/paramsConverter";
+import { SearchResult } from "../interfaces/response/SearchResult";
 import { apiService } from "../apiService";
+import { config } from "../settings";
 
-export interface Category {
-  name: String;
-}
-
-export interface SearchResult<T> {
-  total: Number;
-  data: T[];
-}
+const getCategories = async function(
+  pagination?: any,
+  sort?: any,
+  filter?: any
+): Promise<SearchResult<Category[]>> {
+  const resp = await apiService.get(config.endpoint + getCategoryEndpoint(), {
```

Comment:
Maybe we may use `apiService.defaults.baseURL` instead of `config.endpoint` just like you did in tests? `Config.endpoint` is being used during `resetConfiguration` and it loads baseURL in apiService component.

Reply:
it's deleted now :heavy_check_mark: